yt/frontends/cholla/fields.py

```diff
@@ -0,0 +1,155 @@
+import numpy as np
+from unyt import Zsun
+
+from yt.fields.field_info_container import FieldInfoContainer
+from yt.utilities.physical_constants import kboltz, mh
+
+# Copied from Athena frontend
+pres_units = "code_pressure"
+erg_units = "code_mass * (code_length/code_time)**2"
+rho_units = "code_mass / code_length**3"
+mom_units = "code_mass / code_length**2 / code_time"
+
+
+def velocity_field(comp):
+    def _velocity(field, data):
+        return data["cholla", f"momentum_{comp}"] / data["cholla", "density"]
+
+    return _velocity
+
+
+class ChollaFieldInfo(FieldInfoContainer):
+    known_other_fields = (
```

Comment:
This line is reported by mypy
```
yt/frontends/cholla/fields.py:22: error: Need type annotation for "known_other_fields"  [var-annotated]
```

The fix is this:
```suggestion
    known_other_fields: KnowFieldsT = (
```

(in combination with adding `from yt._typing import KnownFieldsT` at the top level of the module

Reply:
@chummels looks like you forgot the import